src/components/tl/ucp/tl_ucp_context.c

```diff
@@ -133,6 +134,81 @@ ucc_tl_ucp_context_service_init(const char *prefix, ucp_params_t ucp_params,
     return ucc_status;
 }
 
+static int memcpy_device_start(void *dest, void *src, size_t size,
+                                void *completion, void *user_data) {
+
+        ucc_status_t status;
+        ucc_ee_executor_task_args_t eargs;
+        ucc_ee_executor_t *exec;
+        ucc_tl_ucp_task_t *task = (ucc_tl_ucp_task_t *) user_data;
+        // void *non_const_src = (void *) src;
+
+        status = ucc_coll_task_get_executor(&task->super, &exec);
+        if (ucc_unlikely(status != UCC_OK)) {
+            task->super.status = status;
+            return -1;
```

Comment:
maybe u can simply return uuc_status? check its values, it's enum so it's int eventually